slurm_ops_manager/slurm_ops_base.py

```diff
@@ -446,6 +446,25 @@ def setup_nhc(self) -> bool:
 
         return status
 
+    def setup_logrotate(self):
+        """Set up logrotate profiles.
+
+        Copy logorate profiles for NHC and Slurm.
+
+        Returns True on success, False otherwise.
+        """
+        logrotate_dir = Path("/etc/logrotate.d")
+
+        shutil.copyfile(TEMPLATE_DIR / "logrotate_nhc", logrotate_dir / "nhc")
+        shutil.copyfile(TEMPLATE_DIR / "logrotate_slurm", logrotate_dir / "slurm")
+
+        # ubuntu package creates /etc/logrotate.d/slurm{d,dbd,ctld} files, we
+        # need to remove those
+        for daemon in ["slurmd", "slurmdbd", "slurmctld"]:
+            profile = logrotate_dir / daemon
+            if profile.exists():
```

Comment:
you could just use `profile.unlink(missing_ok=True)`

Reply:
No, because this code should work on python3.6 and this flag is not available there:
```python
>>> Path("/tmp/asdfasdf").unlink(missing_ok=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unlink() got an unexpected keyword argument 'missing_ok'
```